src/python/py-hirtectl

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# vim:sw=4:ts=4:et
+
+import argparse
+import argcomplete
+
+from collections import namedtuple
+from hirte import Hirte
+
+
+def hirtectl_node_listall(hirte):
```

Comment:
Shouldn't these all be in the library rather then the client?

Reply:
I would prefer

import hirte
hirte.ListAll()
hirte.Start(units)
hirte.Stop(units)
...

Reply:
Yes, sure we can do in the library but we need a way to expose the data to users so can manipulate the way they want. Maybe someone will want to only use `info.name` data to create `list only my nodes name`.

Reply:
Sure, let's use this naming convention.